torrt/utils.py

```diff
@@ -63,6 +63,22 @@ def dump_contents(filename, contents):
         f.write(text)
 
 
+def base64encode(string_or_bytes):
+    """Return base64 encoded input
+
+    :param string_or_bytes:
+
+    """
+
+    if six.PY3:
+        from base64 import encodebytes
+        string_bytes = string_or_bytes.encode('utf-8') if not isinstance(string_or_bytes, bytes) else string_or_bytes
```

Comment:
Here you not only evalute ``string_or_bytes.encode('utf-8')`` event it may not be necessary, but also use inversion (``if not``).